server/src/__tests__/species.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from '@loopback/testlab';
+import { TreetrackerDataSource } from '../datasources';
+import { SpeciesController } from '../controllers';
+import { SpeciesRepository } from '../repositories';
+require('dotenv').config();
+
+/*Purpose: Include env. variables in test in order to 
+map to env. variables in Github Actions.*/
+describe('Species Controller Integration Tests', () => {
+    //Configure the database with env. variables. 
```

Comment:
While I agree it's useful to be able to configure the DB connection in the environment, the responsibility to read the env vars and set up the datasource object shouldn't sit with the test.